src/patternfly/components/NavSystem/styles.scss

```diff
@@ -0,0 +1,481 @@
+@import "../../patternfly-utilities";
+
+.pf-c-nav {
+  $namespace: &;
+
+  /* stylelint-disable */
+  --pf-c-nav--Width:                                          pf-size-prem(290px);
+  --pf-c-nav--Transition:                                     var(--pf-global--Transition);
+
+  // List link
+  --pf-c-nav__list-link--PaddingTop:                          var(--pf-global--spacer--md);
+  --pf-c-nav__list-link--PaddingRight:                        var(--pf-global--spacer--xl);
+  --pf-c-nav__list-link--PaddingBottom:                       var(--pf-global--spacer--md);
+  --pf-c-nav__list-link--PaddingLeft:                         var(--pf-global--spacer--xl);
+  --pf-c-nav__list-link--FontWeight:                          var(--pf-global--FontWeight--normal);
+  --pf-c-nav__list-link--active--FontWeight:                  var(--pf-global--FontWeight--semi-bold);
+  --pf-c-nav__list-link--not-first-child--PaddingTop:         var(--pf-global--spacer--lg);
```

Comment:
Since this causes the element to have more space at the top than the bottom (so if you use a background color, for example, the text isn't vertically centered), should we keep the padding consistent and add a margin-top instead?

Reply:
Updated